src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -342,6 +342,17 @@ protected final String getSuccessfulDestination() {
         return triggers.toMap();
     }
 
+    public List<TriggerDescriptor> getTriggerDescriptors() {
+        List<TriggerDescriptor> result = new ArrayList<TriggerDescriptor>();
+        for (TriggerDescriptor d: Trigger.for_(this)) {
+            if (d instanceof TimerTrigger.DescriptorImpl) {
+                continue;
```

Comment:
Arguably better done using `DescriptorVisibilityFilter`, so that it would apply automatically to any other UIs offering `Trigger`s here.